opencti-platform/opencti-front/src/private/components/settings/decay/DecayRule.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import { graphql, useFragment } from 'react-relay';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Paper from '@mui/material/Paper';
+import makeStyles from '@mui/styles/makeStyles';
+import DecayChart, { DecayHistory } from '@components/settings/decay/DecayChart';
+import DecayRuleEdition from './DecayRuleEdition';
+import DecayRulePopover from './DecayRulePopover';
+import ExpandableMarkdown from '../../../../components/ExpandableMarkdown';
+import { useFormatter } from '../../../../components/i18n';
+import ItemBoolean from '../../../../components/ItemBoolean';
+import CustomizationMenu from '../CustomizationMenu';
+import { DecayRule_decayRule$key } from './__generated__/DecayRule_decayRule.graphql';
+
+const useStyles = makeStyles(() => ({
```

Comment:
`makeStyles` is deprecated. Try to put style directly in JSX using `style={{...}}` for html elements or `sx={{...}}` for MUI elements